apps/external-devices/device-descriptors/Atmel_ATmega32.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Atmel Corporation",
+    "deviceName": "ATmega32",
```

Comment:
You can certainly add one, that's part of the reason we used JSON - we're not punishing unrecognized properties. I'd certainly suggest a notes field for these - probably standardizing a notes field to generate onto the web site, because for the external devices, where the server isn't built into the vendor's device, the user will want to know what the name of the device is for their `vrpn.cfg` file.